charts/rollups-node/Chart.yaml

```diff
@@ -13,7 +13,7 @@ maintainers:
     url: https://github.com/oap75
 
 type: application
-version: 1.0.0
+version: 1.0.1-0
```

Comment:
the version bump should come together with the version bump for the rollups-node container images

Reply:
AFAIK, we don't have a `1.0.1` yet, but we could use the https://github.com/cartesi/rollups/tree/v1.0.1-rc.0

Reply:
follow the release here https://github.com/cartesi/rollups/pull/238